java/ci/build-in-docker.sh

```diff
@@ -48,9 +48,6 @@ export GIT_COMMITTER_EMAIL="ci@nvidia.com"
 export CUDACXX=/usr/local/cuda/bin/nvcc
 export LIBCUDF_KERNEL_CACHE_PATH=/rapids
 
-# add cmake 3.19 to PATH
-export PATH=/usr/local/cmake-3.19.0-Linux-x86_64/bin:$PATH
```

Comment:
I'd like to move this line into the Dockerfile, where the corresponding version of cmake tar ball is downloaded